Core/include/Acts/Vertexing/IterativeVertexFinder.ipp

```diff
@@ -231,7 +239,7 @@ Acts::IterativeVertexFinder<vfitter_t, sfinder_t>::getCompatibility(
        (vertex.fullCovariance() * linTrack.positionJacobian.transpose()))
           .template block<2, 2>(0, 0);
   weightReduced += errorVertexReduced;
-  weightReduced = weightReduced.inverse();
+  weightReduced = weightReduced.inverse().eval();
```

Comment:
Does this do anything in this case?

Reply:
For some reason I got runtime alignment errors. The error message suggested using eval

Reply:
Strange but ok.